src/test/com/exeptions/MyExeptions.java

```diff
@@ -0,0 +1,46 @@
+package test.com.exeptions;
+
+import java.util.InputMismatchException;
+import java.util.Random;
+import java.util.Scanner;
+
+
+/**
+ * Напиши клас, який виконує наступні дії:
+ * 1. Приймає з клавіатури два цілих числа розділених комою.
+ * 2. Створює і виводить в консоль 20 випадкових чисел у діапазоні від першого, тільки що введеного з клавіатури,
+ * значення до другого значення, також введеного з клавіатури перед цим.
+ * Можна виходити з того, що перше з чисел повинно бути менше другого. Для цього потрібен ексепшн.
+ * Приклад:
+ * Вхід: 1,10    Вихід: 42958147591451412345
+ * Вхід: 5,1      Вихід: IllegalArgumentException “first number should be less than second”
+ * Вхід: %,№   Вихід: IllegalArgumentException “illegal symbols, use only numbers”
+ * Вхід: 1,2,5    Вихід: IllegalArgumentException “maximum 2 numbers”
+ */
+public class MyExeptions {
+    public static void main(String[] args) {
+        final int limitNum = 20;
+        Random rd = new Random();
+        Scanner sc = new Scanner(System.in);
+        String strIn = sc.nextLine();
+
+        String[] strSplit = strIn.split(",");
```

Comment:
splitStr would be more precise. split is a 3rd for of the verb. But you should better name it something like "input" just because it's a bad practice to point the type of the variable in it's name. You don't add anything valuable to naming this way so what is the point. Naming is one of the most important part of development because developers are reading code 80% of the time and if naming is not clear or misleading it can steal a lot of time.